apps/organizations/decorators.py

```diff
@@ -122,3 +124,84 @@ def __call__(self, view_func):
         return require_organization_permission(
             OrganizationMemberRoles.MEMBER, content_type=self.content_type
         )(view_func)
+
+
+class require_organization_member_user(object):
```

Comment:
Not sure if I like `require_` as part of the decorator name prefix for all of these